src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -30,7 +30,33 @@ export class UBAClientWithRefresh extends BaseUBAClient {
     this.relayCalculator = new RelayFeeCalculator(this.relayerConfiguration);
   }
 
+  public async update(): Promise<void> {
+    // Update the Across config store
+    await this.hubPoolClient.configStoreClient.update();
+    // Update the HubPool
+    await this.hubPoolClient.update();
+    // Update the SpokePools
+    await Promise.all(Object.values(this.spokePoolClients).map(async (spokePoolClient) => spokePoolClient.update()));
+    // Update the UBAFeeCalculators
+    await Promise.all(
```

Comment:
We might want to add some logs here for the future when running in production so we can easily see if any of these clients are taking a long time to update and point to some easy optimizations

Reply:
I was hoping to make this a second pass when we're more cemented on the components. Should we add the logging now? Totally happy with it either way